apps/src/lib/levelbuilder/unit-editor/unitEditorRedux.js

```diff
@@ -304,6 +304,22 @@ export const mapLessonGroupDataForEditor = rawLessonGroups => {
   return lessonGroups;
 };
 
+export const mapLessonGroupDataForUpdate = lessonGroups =>
```

Comment:
What is the reason for doing this mapping instead of just sending whatever this.props.lessonGroups is?

Reply:
good catch Dani! there wasn't a great reason for doing this, and our best practice is to send camelCase to the server during ajax calls.